torch/utils/cpp_extension.py

```diff
@@ -173,9 +173,6 @@ def _join_rocm_home(*paths) -> str:
     if ROCM_HOME is None:
         raise OSError('ROCM_HOME environment variable is not set. '
                       'Please set it to your ROCm install root.')
-    elif IS_WINDOWS:
```

Comment:
Since this file is used by both cuda and rocm extensions, be careful removing something like this.  Your intent is to enable this on ROCm Windows but you're also removing this for CUDA.  Leave this OSError specifically for CUDA and avoid it for ROCm.